### PR TITLE
feature/NEP-8-create-GET-request

### PR DESCRIPTION
**Pull request by alex625144**

### CODE REVIEW

File: src/main/java/com/neprozorro/model/ResultReportItem.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "result_report_item")
+public class ResultReportItem {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @Column(name = "Id")
```
Comment (FedosovMax) [inline src/main/java/com/neprozorro/model/ResultReportItem.java:30]: should be id (small letter)

File: src/main/java/com/neprozorro/rest/MockResource.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.neprozorro.service.MockService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController()
```
Comment (FedosovMax) [inline src/main/java/com/neprozorro/rest/MockResource.java:12]: "()" is redundant

File: src/main/java/com/neprozorro/rest/MockResource.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+@RequiredArgsConstructor
+public class MockResource { //Todo for remove
```
Comment (FedosovMax) [inline src/main/java/com/neprozorro/rest/MockResource.java:14]: comment suggests removing but I'm not sure if the comment should be here. Don't see a problem with having an additional controller

File: src/main/java/com/neprozorro/model/ResultReportItem.java (hunk 1 above)
Comment (alex625144) [reply src/main/java/com/neprozorro/model/ResultReportItem.java:30]: Done!

File: src/main/java/com/neprozorro/rest/MockResource.java (hunk 2 above)
Comment (alex625144) [reply src/main/java/com/neprozorro/rest/MockResource.java:12]: Done!

File: src/main/java/com/neprozorro/rest/MockResource.java (hunk 3 above)
Comment (alex625144) [reply src/main/java/com/neprozorro/rest/MockResource.java:14]: Ok.

File: src/main/java/com/neprozorro/rest/ResultResource.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+
+@Slf4j
```
[the remark this answers is not in the document]
Comment (alex625144) [reply src/main/java/com/neprozorro/rest/ResultResource.java:16]: It is true.

**Review by FedosovMax — APPROVED** [review]

**Pull request merged**